distribution/docs/src/main/resources/content/_configuring/admin-wizard-placeholder.adoc

```diff
@@ -0,0 +1,11 @@
+:title: Admin Wizards
+:type: configuration
+:status: published
+:parent: Admin Console Tutorial
+:order: 00
+:summary: Placeholder for review. To be replaced.
+////
+== {title}
+
+TODO: https://codice.atlassian.net/browse/DDF-2193 - update documentation for how to configure to use an LDAP server
+////
```

Comment:
❓ Is this where the new wizards documentations will live when they're merged into DDF or is it just how to configure LDAP?

Reply:
This file is for the wizards specifically, but that ticket number was the closest I had for it.